x-pack/test/security_solution_cypress/cypress/tasks/hosts/events.ts

```diff
@@ -36,3 +36,8 @@ export const opensInspectQueryModal = () => {
 export const waitsForEventsToBeLoaded = () => {
   cy.get(SERVER_SIDE_EVENT_COUNT).should('not.have.text', '0');
 };
+
+export const waitForEventsDataGridToBeLoaded = () => {
+  cy.get('[data-test-subj="events-container-loading-true"]').should('not.exist');
+  cy.get('[data-test-subj="events-container-loading-false"]').should('exist');
```

Comment:
Would it be possible to extract these selectors to the _screens_?
```suggestion
  cy.get(EVENTS_LOADING_TRUE).should('not.exist');
  cy.get(EVENTS_LOADING_FALSE).should('exist');
```

Reply:
Done! https://github.com/elastic/kibana/pull/178890/commits/a8088f4c9553a8c6317b1a928aee077c460ee80c